docs/roadmap.md

```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
+- [ ] Monitoring stack upgrade including Prometheus, Prometheus Operator, Node exporter, kube-state-metrics etc.
+- [ ] Custom metrics support including application custom metrics dashboard, custom metrics HPA
+- [ ] Integration with Alertmanager
+- [ ] K8s Event management
+- [ ] K8s Audit Support
+- [ ] Notification Enhancement
```

Comment:
"Notification Enhancement" is too vague, could you add more detailed description?  e.g. add new notification channels including Wechat...